source/_lovelace/horizontal-stack.markdown

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Horizontal Stack Card"
 sidebar_label: Horizontal Stack
-description: "The Horizontal Stack card allows you to stack together multiple cards, so they always sit next to each other in the space of one column."
+description: "The Horizontal Stack card allows you to stack together multiple cards, so they always sit next to each other in the space of one row."
```

Comment:
This referred to the column in Lovelace. As a row has multiple columns, the horizontal stack cards allow us to fit multiple cards in a single column.

You don't need the horizontal stack card to get multiple cards in a row...

Reply:
Ah. Thanks for clarifying. I almost didn't submit this because I thought, "no way they missed this detail".